pkg/plugins/helm/provider.go

```diff
@@ -196,6 +196,22 @@ func (p *Provider) SplitJobIntoTasks(job *models.Job) (*models.TaskLayer, error)
 			Child: nil,
 		}
 
+		return &tl, nil
+	case constants.ActionUpdateClusterEnv:
+		td := TaskDirective{
+			VersionId:   job.VersionId,
+			RuntimeId:   jodDirective.RuntimeId,
+			Values:      jodDirective.Values,
```

Comment:
jobDirective?

Reply:
JobDirective defined in helm/directive.go

Reply:
typo

Reply:
ok